test/test_suite.py

```diff
@@ -1579,45 +1579,32 @@ def test_user_agent(self):
             )
 
 
-class ExecutionOptionsTest(fixtures.TestBase, unittest.TestCase):
+class ExecutionOptionsReadOnlyTest(fixtures.TestBase):
     """
     Check that `execution_options()` method correctly
     sets parameters on the underlying DB API connection.
     """
 
-    @classmethod
-    def setUpClass(cls):
-        cls._engine = create_engine(get_db_url(), pool_size=1)
-        cls._metadata = MetaData(bind=cls._engine)
+    def setUp(self):
+        self._engine = create_engine(get_db_url(), pool_size=1)
+        metadata = MetaData(bind=self._engine)
 
-        cls._table = Table(
+        self._table = Table(
             "execution_options",
-            cls._metadata,
+            metadata,
             Column("opt_id", Integer, primary_key=True),
             Column("opt_name", String(16), nullable=False),
         )
 
-        cls._metadata.create_all(cls._engine)
+        metadata.create_all(self._engine)
```

Comment:
Splitting the test case into two different cases to avoid some emulator related issues